samples/matter/common/src/bridge/matter_bridged_device.cpp

```diff
@@ -73,6 +74,61 @@ CHIP_ERROR MatterBridgedDevice::HandleReadDescriptor(AttributeId attributeId, ui
 		uint32_t featureMap = GetDescriptorFeatureMap();
 		return CopyAttribute(&featureMap, sizeof(featureMap), buffer, maxReadLength);
 	}
+	case Clusters::Descriptor::Attributes::DeviceTypeList::Id: {
+		return CopyAttribute(mDeviceTypeList, sizeof(mDeviceTypeList), buffer, maxReadLength);
```

Comment:
Have you tested this works correctly? Looking at the `ReadSingleClusterData`, it doesn't appear that lists are supported in the default implementation. Isn't the default implementation of the descriptor cluster supposed to work with bridged endpoints as well (in which case this method wouldn't be needed at all).

Reply:
Well I tested and it worked but it doesn't mean it won't work without this patch :) Will check that.
Btw. @kkasperczyk-no maybe that would explain why the cluster revision of the Descriptor cluster set in the application is always ignored.

Reply:
Also, there is a bug here as I take the wrong size of the list, so this code can't work.

Reply:
You can check the implementation in `descriptor.cpp` but for me it should work for dynamic endpoints as well. If that's the case, you can probably drop all the code you have for the descriptor attribute reads.

Reply:
Descriptor cluster revision seems to be fixed there: `static constexpr uint16_t ClusterRevision = 2;`

Reply:
Yep, the whole code which handles descriptor cluster in the dynamic endpoints is useless. It's handled automatically by the core.